ingest_operations.py

```diff
@@ -392,3 +392,20 @@ def remove_program_for_user(user_id, program_id):
             response, status_code = auth.write_user_in_opa(response, token)
             return response, status_code
     return {"error": f"No program {program_id} found for user"}, status_code
+
+@app.route('/get-token')
+def get_token():
+    # Attempt to grab the token via session_id
+    token = request.cookies['session_id']
+
+    return {"token": token}, 200
```

Comment:
should there be some checks to make sure that `request` has `cookies` or that it has key `session_id`? If I try this endpoint from a direct curl, it'll fail but in a weird way.
```
HTTP/1.1 400 Bad Request
Content-Length: 163
Content-Type: application/problem+json
Date: Sat, 28 Sep 2024 04:19:58 GMT
Server: gunicorn
X-Ratelimit-Limit: 10000
X-Ratelimit-Remaining: 9991
X-Ratelimit-Reset: 1727500376
Connection: close

{
  "detail": "The browser (or proxy) sent a request that this server could not understand.",
  "status": 400,
  "title": "Bad Request",
  "type": "about:blank"
}
```